{
  "modified": "2025-07-31T19:38:32Z",
  "published": "2025-07-31T19:38:32Z",
  "schema_version": "1.5.0",
  "id": "",
  "aliases": [
    "SNYK-JS-PYDANTIC-10335833",
    "MAL-2025-4867",
    "GHSA-cmwc-w4x6-hx68"
  ],
  "affected": [
    {
      "package": {
        "ecosystem": "npm",
        "name": "pydantic"
      }
    }
  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://security.snyk.io/vuln/SNYK-JS-PYDANTIC-10335833"
    },
    {
      "type": "PACKAGE",
      "url": "https://npmjs.com/package/pydantic"
    },
    {
      "type": "ADVISORY",
      "url": "https://github.com/advisories/GHSA-cmwc-w4x6-hx68"
    },
    {
      "type": "ADVISORY",
      "url": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/npm/pydantic/MAL-2025-4867.json"
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "source": "reversing-labs",
        "sha256": "784ff69d7b92064310e2ef7a3016943dcd8a06cfce7cca647a7b9be8f2d786ff",
        "import_time": "2025-08-01T10:41:40.527930471Z",
        "id": "RLUA-2025-03994",
        "modified_time": "2025-07-31T19:38:32Z"
      }
    ]
  }
}
